packages/utils/crates/utils-types/src/primitives.rs

```diff
@@ -362,6 +362,15 @@ impl Snapshot {
             log_index,
         }
     }
+
+    #[cfg_attr(feature = "wasm", wasm_bindgen::prelude::wasm_bindgen)]
+    pub fn zero() -> Self {
```

Comment:
Rust callers should use `Snapshot::default()`. Let's move this function completely into the `pub mod wasm` block and make it call `Self::default()` internally (see L345)

Reply:
Implementing the `Default` disallows it from being used in a `wasm-bindgen` context. Should the `Default` trait be dropped and left as `default` only? Or a custom function that mimics the behavior? Is `zero` incorrect? All of the members are zero.

Reply:
No I mean keeping `Default` impl in the code, but removing the `zero()` method here and moving it into `impl Snapshot` under the `mod wasm`. That way the `zero()` method will be accessible from JS, but pure Rust would used `default()`